chroma-manager/requirements.dev

```diff
@@ -18,4 +18,4 @@ pyflakes
 pygraphviz
 sphinx==1.1.3
 werkzeug
-iml-common
+iml-common>=1.0.5
```

Comment:
we need to pin here because the unit tests don't clear packages inside the virtual environment used, if you want to resolve that it would be much appreciated (as per previous discussion)